src/Kovan-DssSpell.sol

```diff
@@ -15,71 +15,160 @@
 // along with this program.  If not, see <https://www.gnu.org/licenses/>.
 pragma solidity 0.6.11;
 
+import {Fileable, ChainlogLike} from "dss-exec-lib/DssExecLib.sol";
 import "dss-exec-lib/DssExec.sol";
 import "dss-exec-lib/DssAction.sol";
+import "dss-interfaces/dss/IlkRegistryAbstract.sol";
+
+interface ClipperMomLike {
+    function setAuthority(address) external;
+}
 
 import "dss-interfaces/dss/FaucetAbstract.sol";
 
 contract DssSpellAction is DssAction {
 
     string public constant description = "Kovan Spell";
 
+    address constant MCD_DOG              = address(0);
+    address constant MCD_END              = address(0);
+    address constant MCD_ESM              = address(0);
+    // address constant ILK_REGISTRY         = address(0);
+    address constant CLIPPER_MOM          = address(0);
+    address constant MCD_CLIP_LINK_A      = address(0);
+    address constant MCD_CLIP_CALC_LINK_A = address(0);
```

Comment:
I'm not really sure if all collaterals will be using the same params, so I think it will be much more flexible doing this way. Worst case we set multiple entries with the same address. For example now `PIP_WBTC` is the same than `PIP_RENBTC`.